assets/scss/pages/privacy_questions.scss

```diff
@@ -0,0 +1,7 @@
+.dpa-types {
+  .help-button {
+    position: revert;
```

Comment:
oof, postition revert sounds complicated. Did you come up with this, or is it a copy paste?
Why not use initial or the position you want? Now you are making it dependent on the parent.

Just wondering